assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -118,20 +127,22 @@ const CloseButton = styled(Button)`
 const SnackbarMessage = ({
   actionLabel,
   handleAction = () => {},
-  handleDismiss,
+  handleDismiss = () => {},
   isPreventAutoDismiss,
+  isPreventActionDismiss,
   message,
   removeMessageTimeInterval,
   showCloseButton,
+  place = 'bottom',
```

Comment:
Maybe it would be good to rename this to `placement` for consistency with the other components?